azurelinuxagent/common/osutil/default.py

```diff
@@ -76,6 +79,31 @@ def __init__(self):
         self.selinux = None
         self.disable_route_warning = False
 
+    def get_firewall_dropped_packets(self, dst_ip=None):
+        global _enable_firewall
+        if not _enable_firewall:
+            return 0
+
+        try:
+            wait = self.get_firewall_will_wait()
+
+            rc, output = shellutil.run_get_output(FIREWALL_PACKETS.format(wait))
+            if rc != 0:
+                return -1
+
+            pattern = re.compile(PACKET_PATTERN.format(dst_ip))
+            for line in output.split('\n'):
+                m = pattern.match(line)
+                if m is not None:
+                    return int(m.group(1))
+            
+            return 0
+
+        except Exception as e:
```

Comment:
There was an issue logged where the agent was spamming the logs because the VM did not have iptables installed.  (I vaguely remember CoreOS).  If we can determine iptables is not available I would not do this and thereby avoid the log spamming.

Reply:
It may already be Ok, but I'll review closely.